tools/src/launcher/Main.hs

```diff
@@ -96,6 +105,10 @@ data LauncherOptions = LO
 -- | The concrete monad where everything happens
 type M a = (HasConfigurations, HasCompileInfo) => Log.LoggerNameBox IO a
 
+data LType = LInfo | LNotice | LWarning | LError
```

Comment:
AFAIU, only only `LInfo` and `LError` are used. Do we need other two? If we want it to contain only those log types which we are going to use, then let's have only two constructors. If we want all possible log types, then why not just use `Severity` type from `log-warper`?

Reply:
Done with two constructors, I don't think we need more.